ios/Podfile

```diff
@@ -18,7 +18,11 @@ target 'StatusIm' do
   # Pods for StatusIm
   pod 'Instabug', '~> 7.0'
   pod 'FirebaseMessaging'
-  
+
+  pod 'React', :path => '../node_modules/react-native'
+  pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga'
```

Comment:
Can you detail why those changes?

Reply:
It was added automatically when I ran `react-native link` after adding `react-native-background-timer`. I believe it's required by `React/Core` as a dependency. And `react-native-background-timer` needs `React` as a dependency, so it ended up having to add `React` > `React/Core` > `yoga`.